bin/reportgenerator.properties

```diff
@@ -34,6 +34,8 @@
 # Reporting configuration
 #---------------------------------------------------------------------------
 
+jmeter.reportgenerator.msns=ms
```

Comment:
`jmeter.reportgenerator.msns` is a strange property name. Consider someone would need `minutes` later. I would suggest something like `jmeter.reportgenerator.time.unit=ns|ms|minutes|...`.

However,